proto/identity.go

```diff
@@ -1,10 +1,17 @@
 package proto
 
 import (
+	"encoding/json"
+	"io/ioutil"
+	"regexp"
 	"strings"
 	"unicode/utf8"
 )
 
+var validEmoji = make(map[string]string)
```

Comment:
Generally, only use make() if there is no alternative literal constructor (e.g., making a slice of non-zero size or capacity).

```
var validEmoji = map[string]string{}
```